packages/pwa/app/utils/utils.js

```diff
@@ -162,3 +163,153 @@ export const capitalize = (text) => {
  * @returns {object} - url object from the pwa-kit-config.json file
  */
 export const getUrlConfig = () => pwaKitConfig?.app?.url
+/**
+ * A util to return current multi site configuration
+ * @returns {object} - a list of site objects from the pwa-kit-config.json file
+ */
+export const getSitesConfig = () => pwaKitConfig?.app?.sites
+/**
+ * A util to return current defaultSiteId
+ * @returns {string} - default site Id string from pwa-kit-config file
+ */
+export const getDefaultSiteId = () => pwaKitConfig?.app?.defaultSiteId
+
+/**
+ * Return a the configuration of the given key from pwa-kit-config
+ * By default, returns app configuration
+ * @param keys {array} - array of keys
+ * @returns {object}
+ */
+export const getConfig = () => pwaKitConfig
+
+export const getSite = (url) => {
```

Comment:
Can you do the following before I start reviewing the contents of this function:

1. Write some JSDOC for this function, describing what it takes as parameters and what it returns.
2. I'm guessing that this is your site resolution function, it would be nice to have some inline comments to help the reader (me) understand what you are doing at each step.
3. The function name is `getSite` that takes a `url`.. you also have a function called `getSiteByUrl` that takes a `url`, you can see that there might be some confusion on what makes these 2 functions different.